### PR TITLE
MyBike landing page

### PR DESCRIPTION
**Pull request by FelixTheFallen**
[DEMO LINK](https://FelixTheFallen.github.io/layout_landing-page/)

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your solution

**Comment by FelixTheFallen** [comment]
Uh sorry for a question but how to deploy work?

**Comment by FelixTheFallen** [comment]
src/index.html
[error] src/index.html: SyntaxError: Opening tag "link" not terminated. (11:5)
[error]    9 |     <link rel="preconnect" href="https://fonts.gstatic.com">
[error]   10 |     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
[error] > 11 |     <link rel="stylesheet" href="styles/block/main.scss"
[error]      |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[error] > 12 |     <script src="scripts/main.js" defer></script>
[error]      | ^^^^^
[error]   13 |   </head>
[error]   14 |   <body class="page__body">
[error]   15 |     <header class="header">

and what this error means?

File: src/index.html [hunk 1]
```diff
@@ -1,22 +1,152 @@
...
-      defer
-    ></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyBike</title>
+    <link rel="icon" type="image/icon" href="images/MyBikeLogo.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/block/main.scss"
```
Comment (volodymyr-soltys97) [inline src/index.html:11]:
```suggestion
    <link rel="stylesheet" href="styles/block/main.scss">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, the demo link isn't working, you need to make a deploy and check it after on the demo link 
image

**Pull request closed without merging**